### ISSUE TITLE
is it possible to run more pool than 1, if so can someone please put the modified text for coins.json here for a sample

### ISSUE THREAD

**Issue by david739** [oliverw/miningcore#1548]
is it possible to run more pool than 1, if so can someone please put the modified text for coins.json here for a sample

**Comment by oliverw** [comment]
Did you mean `config.json` ?

**Comment by david739** [comment]
good day yes

**Comment by seperatis** [comment]
@david739 Yes, you can run many pools with 1 instance of MiningCore. This is a question, not a software BUG. Therefore I am converting this into a Discussion. 

In the future, please only open a new Issue for problems related directly to the code itself. That is the purpose of opening new Issue; bugs, performance problems, etc. Not for feature requests, not for general questions, not for training you on how to use linux or database software, etc.

Thanks.
